client/ayon_nuke/api/plugin.py

```diff
@@ -438,16 +440,24 @@ def _get_render_target_enum(self):
             "local": "Local machine rendering",
             "frames": "Use existing frames"
         }
-        if "farm_rendering" in self.instance_attributes:
-            rendering_targets.update({
-                "frames_farm": "Use existing frames - farm",
-                "farm": "Farm rendering",
-            })
+
+        farm_rendering_targets = {
+            "frames_farm": "Use existing frames - farm",
+            "farm": "Farm rendering",
+        }
+
+        if (
+            "farm_rendering" in self.instance_attributes
+            or self.render_target in farm_rendering_targets
+        ):
+            rendering_targets.update(farm_rendering_targets)
```

Comment:
I don't think we should add farm options just because you have farm value set in settings as default -> it's hiding wrong settings.
```suggestion
        if "farm_rendering" in self.instance_attributes:
            rendering_targets.update({
                "frames_farm": "Use existing frames - farm",
                "farm": "Farm rendering",
            })
```

Reply:
On the other end, wouldn't it be confusing for the user to be able to set `Farm rendering` in the settings but not seing it applied and available because the `farm_rendering` attribute is missing ?

or I can make it raises with a `CreatorError` if both options conflicts.